addons/l10n_jo_edi/tests/jo_edi_common.py

```diff
@@ -0,0 +1,100 @@
+from odoo import Command
+from odoo.tools import misc
+from odoo.addons.account_reports.tests.common import TestAccountReportsCommon
```

Comment:
@hsal-odoo `account_reports` is not a dependency, it's an enterprise module, can you update the test and make sure it runs with module dependencies only?

Reply:
Apparently it can just be replaced by AccountTestInvoicingCommon.setup_country